src/gmpv_control_box.c

```diff
@@ -446,6 +450,9 @@ static void gmpv_control_box_init(GmpvControlBox *box)
 	init_button(	box->previous_button,
 			"media-skip-backward-symbolic",
 			_("Previous Chapter") );
+	init_button(	box->repeat_button,
+      "media-playlist-repeat",
```

Comment:
```suggestion
			"media-playlist-repeat-symbolic",
```
Use tabs to indent. Also, explicitly use symbolic variant of the icon.